drivers/bus/acpi/compbatt/compbatt.c

```diff
@@ -20,31 +20,33 @@ ULONG CompBattDebug;
 
 NTSTATUS
 NTAPI
-CompBattOpenClose(IN PDEVICE_OBJECT DeviceObject,
-                  IN PIRP Irp)
+CompBattOpenClose(
+    _In_ PDEVICE_OBJECT DeviceObject,
+    _In_ PIRP Irp)
 {
     PAGED_CODE();
-    if (CompBattDebug & 0x100) DbgPrint("CompBatt: ENTERING OpenClose\n");
+    if (CompBattDebug & COMPBATT_DEBUG_TRACE) DbgPrint("CompBatt: ENTERING OpenClose\n");
```

Comment:
It seems you could simplify such code by introducing a set of macros in the header:
```c
#define TRACE(fmt, ...) \
    do { if (CompBattDebug & COMPBATT_DEBUG_TRACE) DbgPrint(fmt, ##__VA_ARGS__) } while(0)
```
and similar for INFO, WARN and ERR.
And maybe these macros could also be defined to nops in case we're not compiling in DBG mode:
```c
#if DBG
    #define TRACE(fmt, ...) // See proposed definition above
    // INFO, WARN, ERR
#else
#if defined(_MSC_VER)
    #define TRACE(...)    __noop
    // INFO, WARN, ERR
#else
    #define TRACE(...) do { if(0) DbgPrint(__VA_ARGS__); } while(0)
    // INFO, WARN, ERR
#endif
#endif
```

Reply:
That's a great idea indeed, but I feel any miscellaneous improvements to the driver shall be posted in subsequent PRs. There's already a PR of Justin which is currently on-hold...